### ISSUE TITLE
Redesign: room header

### ISSUE THREAD

**Issue by bwindels** [vector-im/element-web#7160]
- [x] new icons
 - [x] make edits work
 - [x] fix content editable layout issues
 - [ ] support expanding the header to show long topics
 - [ ] reskin

Would probably make sense to postpone expanding header this till after #7581 is done, to avoid double work.

**Comment by uhoreg** [comment]
"support expanding the header to show long topics" will fix https://github.com/vector-im/riot-web/issues/1836

**Comment by bwindels** [comment]
@nadonomy I'm wondering what the expanded topic should look like. If we just make the whole header higher, that might take up a lot of space, as the room header is now above the right panel.

**Comment by robintown** [comment]
The room header has been redesigned separately from this, and https://github.com/vector-im/element-web/issues/9623 exists to track the 'topic expansion' task.
